.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v8
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          stale-pr-message: 'This PR is stale because it has been open 45 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
+          close-issue-message: 'This issue was closed because it has been stalled for 5 days with no activity.'
+          close-pr-message: 'This PR was closed because it has been stalled for 10 days with no activity.'
+          days-before-issue-stale: 30
+          days-before-pr-stale: 45
+          days-before-issue-close: 5
+          days-before-pr-close: 10
```

Comment:
![image](https://github.com/microsoft/promptflow/assets/47586720/371267be-de9a-4ed9-8e78-f1fb84742a28)
we might refine the wording following this. Will the auto message at the contributor?

Reply:
I think it's doable, I notice there are some macros seems good to this.

Reply:
maybe I can do some test in my repo first, I think we cannot afford touching all issues in this repo with unpredict behavior

Reply:
have tested, unfortunately, the mention is a feature implemented inside Azure SDK tools: may take a glance at [this](https://github.com/Azure/azure-sdk-tools/blob/1e06e970fa28551411ea2ce1f596273d4ef14275/tools/github-event-processor/RULES.md?plain=1#L329)